README.md

```diff
@@ -68,15 +68,27 @@ update:
     labels: ["WIP", "Update Me"]
 ```
 
-### Caveats and Notes
-
-If both `blacklist` and `whitelist` are specified, bulldozer will attempt to match on both. 
-In cases where both match, `blacklist` will take precedence.
-
-The `merge_method` specifies the strategy that will be used to merge. Possible choices
-are `merge`, `squash`, and `rebase`. Specifying `squash` will allow for a further
-set of `squash_strategy` options, `pull_request_body`, `summarize_commits` and
-`empty_body` that will constitute the body of the merge commit message. 
+## FAQ
+
+#### Can I specify both a `blacklist` and `whitelist`?
+Yes. If both `blacklist` and `whitelist` are specified, bulldozer will attempt to match
+on both. In cases where both match, `blacklist` will take precedence.
+
+#### Can I specify the body of the commit when using the `squash` strategy?
+Yes. Specifying `squash` will allow for a further set of `squash_strategy` options,
+specifically how to render the commit `body`. The possible options are:
+`pull_request_body`, `summarize_commits` and `empty_body`.
+
+#### Bulldozer isn't merging my commit when it should, what could be happening?
+Bulldozer will attempt to merge a branch whenever it passes the whitelist/blacklist
+criteria. GitHub may prevent it from merging a branch in certain conditions, some of
+which are to be expected, and others that may be caused by mis-configuring bulldozer.
+
+* All required Status Checks have passed
+* All PR review requirements are satisfied
+* The merge strategy configured in `.bulldozer.yml` is allowed in your repository settings
+* Branch protection rules are preventing `bulldozer [bot]` from [pushing to the branch](https://help.github.com/articles/about-branch-restrictions/).
```

Comment:
The first 3 are conditions that must be true for a merge, while this last one is something that must not be true. Can you phrase all of them in the same way? Probably the negative, since the intro sentence says "prevent it from merging", e.g.:

- Required status checks have not passed
- Review requirements are not satisfied
- ...